docs/api.rst

```diff
@@ -40,11 +40,11 @@ The code below shows how to execute a complete publish using this API:
     manager.collect_files([path1, path2, path3])
 
     # validate the items to publish
-    items_failed_validation = manager.validate()
+    tasks_failed_validation = manager.validate()
```

Comment:
Since validate now returns the list of tasks that failed, we have to update this part of the doc.